src/math/transform.rs

```diff
@@ -1,8 +1,8 @@
 use std::ops::{Mul, MulAssign};
 
-use glam::{EulerRot, Mat3, Mat4, Quat, Vec2, Vec3, Vec4};
+use glam::{EulerRot, Mat3, Mat4, Quat, Vec2, Vec3};
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Copy, Debug)]
 pub struct Transform {
```

Comment:
Are you sure it's ok to make the `Transform` copy? It seems like it's already a heavy struct

Reply:
Hmmm ideally we shouldn't copy `Transform` that much yeah, but idk if making if `!Copy` is the way to achieve that.